Header header
float64 posX
float64 posY
float64 velX
float64 velY